fhir-server-webapp/src/main/liberty/config/bootstrap.properties

```diff
@@ -1,3 +1,5 @@
 javax.xml.stream.XMLInputFactory=com.sun.xml.internal.stream.XMLInputFactoryImpl
 javax.xml.stream.XMLOutputFactory=com.sun.xml.internal.stream.XMLOutputFactoryImpl
 javax.xml.stream.XMLTransformerFactory=com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl
+
+jaxrs.cxf.use.noop.requestPreprocessor=true
```

Comment:
```suggestion
# Required to avoid conflict between CXF and FHIR interpretation of the _type query param
jaxrs.cxf.use.noop.requestPreprocessor=true
```